docs/guides/finetuning-models.md

```diff
@@ -19,7 +19,7 @@ To summarize:
 `finetunedModel = training_image(baseModel, trainingDataset)`. 
 
 
-The following Model resource would create a finetuned model:
+The following./guides/finetuning-models.ipynb Model resource would create a finetuned model:
```

Comment:
this seems like a copy pasta gone wrong?

Reply:
Fixed